pkg/kube/inject/webhook_test.go

```diff
@@ -1524,3 +1527,12 @@ func BenchmarkInjectServe(b *testing.B) {
 		wh.serveInject(httptest.NewRecorder(), req)
 	}
 }
+
+// defaultInstallPackageDir returns a path to a snapshot of the helm charts used for testing.
+func defaultInstallPackageDir() string {
+	wd, err := os.Getwd()
+	if err != nil {
+		panic(err)
+	}
+	return filepath.Join(wd, "../../../operator/cmd/mesh/testdata/manifest-generate/data-snapshot")
```

Comment:
don't use the snapshot

Reply:
if we are making that argument that the webhook tests are checking webhook logic rather than template changes, why would we continue to use live charts here? you will keep getting the golden conflicts that people complain about.

Reply:
Good point. I went overboard on the comments :slightly_smiling_face: